bundles/org.openhab.binding.boschshc/src/main/java/org/openhab/binding/boschshc/internal/services/privacymode/PrivacyModeState.java

```diff
@@ -0,0 +1,55 @@
+/**
+ * Copyright (c) 2010-2022 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.boschshc.internal.services.privacymode;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.openhab.core.library.types.OnOffType;
+
+/**
+ * Possible privacy mode states of security cameras.
+ * 
+ * @author David Pace - Initial contribution
+ *
+ */
+@NonNullByDefault
+public enum PrivacyModeState {
+
+    /**
+     * Privacy mode enabled / camera disabled
+     */
+    ENABLED,
+
+    /**
+     * Privacy mode disabled / camera enabled
+     */
+    DISABLED;
```

Comment:
The API exposes the so called _privacy mode_, which is the opposite of "camera enabled". With the current implementation we expose the API 1:1, meaning if the user switches the privacy mode off, the camera will be on. This might be counterintuitive for the uses, who probably expect that the camera is enabled when the switch is on.

Do you think we should rather offer an inverse logic channel/state? How would you call this channel? Maybe `camera-state`?

Reply:
I would stay as close to the official API as possible, so keeping the privacy mode here. It may include more than the camera deactivation in the future, maybe it already does. You could add a read-only `camera-state` channel of course which inverts this flag. But writing it should only be possible via the `privacy-mode` channel.

Reply:
I decided to go only for the `privacy-mode` channel in the initial version that exposes the API 1:1. If required, other channels can be added later.